apps/govern/common-util/config/wagmi.ts

```diff
@@ -13,22 +13,13 @@ import {
 } from 'wagmi/chains';
 import { coinbaseWallet, injected, safe, walletConnect } from 'wagmi/connectors';
 
-import { virtualGnosis, virtualMainnet, virtualPolygon, virtualMode } from '../../tenderly.config';
 import { RPC_URLS } from 'libs/util-constants/src';
 
-const mainnetChain =
```

Comment:
removed this after Josh's proposal in a previous PR, but not completely - imo it would be easier to switch between prod and test RPCs by simply updating `NEXT_PUBLIC_IS_CONNECTED_TO_TEST_NET` (once we have a paid tenderly account); however, there might not be a need for this, let's see

Reply:
yeah, i agree, or read NODE_ENV development~
..._TEST_NET is a tough one, as there are testnets like Sepolia, Goerli, .etc
either way, thanks for addressing 🙏